scripts/pi-hole/php/footer.php

```diff
@@ -36,34 +36,69 @@
+    require "scripts/pi-hole/php/update_checker.php";
+?>
+
 <?php
-  // Flushes the system write buffers of PHP. This attempts to push everything we have so far all the way to the client's browser.
-  flush();
-  // Run update checker
-  //  - determines local branch each time,
-  //  - determines local and remote version every 30 minutes
-  require "scripts/pi-hole/php/update_checker.php";
+    $coreVersionStr = $core_current . (isset($core_commit) ? " (" . $core_branch . ", " . $core_commit . ")" : "");
+    $webVersionStr = $web_current . (isset($web_commit) ? " (" . $web_branch . ", " . $web_commit . ")" : "");
+    $ftlVersionStr = $FTL_current . (isset($FTL_commit) ? " (" . $FTL_branch . ", " . $FTL_commit . ")" : "");
+
+    $githubBaseUrl = "https://github.com/pi-hole";
+    $coreUrl = $githubBaseUrl . "/pi-hole";
+    $webUrl = $githubBaseUrl . "/AdminLTE";
+    $ftlUrl = $githubBaseUrl . "/FTL";
+
+    $coreReleasesUrl = $coreUrl . "/releases";
+    $webReleasesUrl = $webUrl . "/releases";
+    $ftlReleasesUrl = $ftlUrl . "/releases";
 ?>
-    <!-- /.content-wrapper -->
     <footer class="main-footer">
-	<!-- Version Infos -->
-        <div class="pull-right hidden-xs hidden-sm<?php if(isset($core_commit) || isset($web_commit) || isset($FTL_commit)) { ?> hidden-md<?php } ?>">
-            <b>Pi-hole Version </b> <?php
-            echo $core_current;
-            if(isset($core_commit)) { echo " (".$core_branch.", ".$core_commit.")"; }
-            if($core_update){ ?> <a class="alert-link lookatme" href="https://github.com/pi-hole/pi-hole/releases" rel="noopener" target="_blank">(Update available!)</a><?php } ?>
-            <b>Web Interface Version </b><?php
-            echo $web_current;
-            if(isset($web_commit)) { echo " (".$web_branch.", ".$web_commit.")"; }
-            if($web_update){ ?> <a class="alert-link lookatme" href="https://github.com/pi-hole/AdminLTE/releases" rel="noopener" target="_blank">(Update available!)</a><?php } ?>
-            <b>FTL Version </b> <?php
-            echo $FTL_current;
-            if(isset($FTL_commit)) { echo " (".$FTL_branch.", ".$FTL_commit.")"; }
-            if($FTL_update){ ?> <a class="alert-link lookatme" href="https://github.com/pi-hole/FTL/releases" rel="noopener" target="_blank">(Update available!)</a><?php } ?>
+        <div class="row row-centered text-center">
+            <div class="col-xs-12 col-sm-6">
+                <strong><a href="https://pi-hole.net/donate/" rel="noopener" target="_blank"><i class="fa fa-heart text-red"></i> Donate</a></strong> if you found this useful.
+            </div>
+        </div>
+
+        <div class="row row-centered text-center version-info">
+            <div class="col-xs-12 col-sm-8 col-md-6">
+                <?php if (isset($core_commit) || isset($web_commit) || isset($FTL_commit)) { ?>
+                <ul class="list-unstyled">
+                    <li><strong>Pi-hole Version</strong> <?php echo $coreVersionStr; ?></li>
```

Comment:
We could probably skip the "Version" part?

Reply:
I'm thinking along the lines of: (edited in the browser)

![image](https://user-images.githubusercontent.com/1998970/82724648-7a2aa580-9ccf-11ea-8d94-387ab91a88c8.png)

@DL6ER / @dschaper - any preference to add here?

Reply:
Does this PR make the footer taller? It currently is a one-liner for me
![Screenshot from 2020-05-23 10-09-59](https://user-images.githubusercontent.com/16748619/82725487-a1886f00-9cdd-11ea-82e7-d3e7825d9d46.png)

Reply:
Yes, it does for non-stable tags.

Reply:
So, @DL6ER what's your take about dropping the "Version" part? It seems a little redundant after discussing it with @PromoFaux

Reply:
I agree